### DISCUSSION TITLE
Lots of links in docs are broken

### DISCUSSION THREAD

**Discussion by blackgrouse** [artilleryio/artillery#1039] [Artillery]
Hi

I'm trying to learn artillery and it's being made really hard because lots of the hyperlinks in the [Test Script Reference](https://artillery.io/docs/guides/guides/test-script-reference.html) are broken.  They either get a 404 from S3 as the bucket doesn't exist or the links to other sections do nothing.  Please fix.

**Comment by hassy** [comment]
Thank you @blackgrouse, should be fixed now

**Reply by blackgrouse** [reply]
Looks good now, thank you @hassy !
